src/controlobjectscript.cpp

```diff
@@ -0,0 +1,45 @@
+#include <QApplication>
+#include <QtDebug>
+
+#include "controlobjectscript.h"
+
+ControlObjectScript::ControlObjectScript(const ConfigKey& key, QObject* pParent)
+        : ControlObjectSlave(key, pParent) {
+}
+
+ControlObjectScript::~ControlObjectScript() {
```

Comment:
Do we really need this empty destructor. ControlObjectScript is already derived from a base class with a virtual destructor, so we don't even need to declare one.